darts/models/forecasting/forecasting_model.py

```diff
@@ -1403,7 +1409,7 @@ def _get_encoders_n(self, n) -> int:
         return n
 
 
-class FutureCovariatesLocalForecastingModel(LocalForecastingModel, ABC):
+class FutureCovariatesLocalForecastingModel(LocalMultivariateForecastingModel, ABC):
```

Comment:
I don't like this class structure too much, because it means that local future covariates models must necessarily support multivariate series. Whereas we could in principles have some future covariates models working on univariate series only (although we maybe don't have any at the moment).

I think this one should still inherit from `LocalForecastingModel` only, WDYT?
Also a cleaner approach might be to rethink most of our model super-classes and see them as "mix-in interfaces" more. But that would be beyond the scope of this PR :)

Reply:
I thought about the mixin approach but it seemed out of scope. I thought that requiring support for multivariate series was ok because that's how it was done previously. By default all models that inherit from `LocalForecastingModel` were checked if they are fitted on univariate time series with explicit if statement that excluded `FutureCovaraitesLocalForecastingModel`. The only difference with this approach is that I don't exclude that specific class but all models that are in `LocalMultivaraiteForecastingModel`.